internal/staticanalysis/parsing/js/babel-parser.js

```diff
@@ -25,150 +26,165 @@ function logJSON(type, subtype, name, pos, array, extra = null) {
     parseOutputLines.push(json);
 }
 
-function logIdentifierJSON(subtype, name, pos, extra = null) {
-    logJSON("Identifier", subtype, name, pos, null, extra);
-}
+function logIdentifierOrPrivateName(identifierType, node) {
+    // if node is a PrivateName, the corresponding Identifier can be found as node.id
+    let identifierNode;
+    switch (node.type) {
+        case "Identifier":
+            identifierNode = node;
+            break;
+        case "PrivateName":
+            identifierNode = node.id;
+            break;
+        default:
+            console.log("Error: logIdentifierNodeJSON passed a node of type " + node.type);
+            return;
+    }
+
+    let name = identifierNode.name;
+    let pos = locationString(identifierNode)
+
+    if (identifierNode.name === undefined) {
+        console.log("Error: undefined identifier name at pos " + pos);
+    }
 
-function logLiteralJSON(subtype, value, pos, inArray, extra = null) {
-    logJSON("Literal", subtype, value, pos, inArray, extra);
+    logJSON("Identifier", identifierType, name, pos, null, null);
 }
 
-function logParameter(p) {
-    if (p === null) {
+function logLiteral(literalType, value, pos, inArray, extra = null) {
+    if (value === undefined) {
+        console.log("Error: undefined literal value at pos " + pos);
         return;
     }
-    if (p.type === "Identifier") {
-        // simple function parameter name: function f(x)
-        logIdentifierJSON("Parameter", p.name, locationString(p));
-    } else if (p.type === "AssignmentPattern" && p.left.type === "Identifier") {
-        // parameter with default value: function f(x = 3)
-        logIdentifierJSON("Parameter", p.left.name, locationString(p));
-    }
+    logJSON("Literal", literalType, value, pos, inArray, extra);
 }
 
-function logParameters(node) {
-    for (let p of node.params) {
-        logParameter(p);
+function visitIdentifierOrPrivateName(path) {
+    const node = path.node;
+    const parentNode = path.parentPath.node;
+    const parentParentNode = path.parentPath.parentPath.node;
+
+    switch (parentNode.type) {
+        case "ObjectProperty":
+        // fall through
```

Comment:
nit: comment not relevant ?

(also indent is wrong).

Reply:
Will fix in another PR